src/components/codeContext.tsx

```diff
@@ -1,5 +1,7 @@
 import {createContext, useEffect, useState} from 'react';
 
+type CodeContextStatus = 'LOADING' | 'LOADED';
```

Comment:
nit: shouldn't this be lowercase?

Reply:
good question, not sure about the conventions there 🤔 But I guess you're right, I'll make it lowercase! 👍

Reply:
Why didn't we just make this `isLoading`?

Reply:
I initially thought about wanting `error` state or something like this as well 🤔 ended up ignoring this for now. I can adjust this a bit.